src/styles/untils/_reset.scss

```diff
@@ -0,0 +1,17 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
I think it should be moved to normalize.scss

Reply:
agree, move that to normalize